tests/ui_tests/app/page/common/table/page.py

```diff
@@ -13,6 +13,8 @@
 class CommonTableObj(BasePageObject):
     """Class for manipulating with common tables elements."""
 
+    loading_state_text = 'autorenew'
```

Comment:
If it's constant, uppercase can clarify it

Reply:
changed